apps/tailwind-components/components/table/TableEMX2.vue

```diff
@@ -64,67 +64,61 @@ function handlePagingRequest(page: number) {
     >
     </FilterSearch>
   </div>
-  <div class="overflow-x-auto overscroll-x-contain">
-    <table class="text-left table-fixed w-full">
-      <caption class="caption-bottom pt-8 text-left">
-        <span class="sr-only">{{ tableId }}</span>
-        Showing
-        {{
-          rows.length
-        }}
-        of
-        {{
-          count
-        }}
-      </caption>
-      <thead>
-        <tr>
-          <th
-            v-for="column in columns"
-            class="py-2.5 px-2.5 border-b border-gray-200 first:pl-0 last:pr-0 sm:first:pl-2.5 sm:last:pr-2.5 text-left w-64"
-            :ariaSort="
-              settings.orderby.column === column.id
-                ? mgAriaSortMappings[settings.orderby.direction]
-                : 'none'
-            "
-            scope="col"
-          >
-            <button
-              class="overflow-ellipsis whitespace-nowrap overflow-hidden hover:cursor-pointer"
-              @click="handleSortRequest(column.id)"
+
+  <div class="overflow-auto rounded-b-50px">
+    <div class="overflow-x-auto overscroll-x-contain bg-table rounded-t-3px">
```

Comment:
To have enough spacing at the bottom, it's possible to add `pb-*` here. In this design, I noticed that the text hugs the left side of the table. We could consider adding a little bit of space in the first cell in each row.